### DISCUSSION TITLE
Référencement de page pré-remplie de type "2600€ en net"

### DISCUSSION THREAD

**Discussion by mquandalle** [betagouv/mon-entreprise#2681] [Idées]
Il y a un volume de recherche important pour des requêtes du type « [2600€ brut en net](https://www.google.com/search?q=2600+brut+en+net) ». Les résultats qui sortent en premier sont des pages incluant le montant précis rendu côté serveur.

On peut déjà répondre à ce type de questions mais [nos pages correspondantes](https://mon-entreprise.fr/simulateurs/salaire-brut-net?salaire-brut=2600%E2%82%AC%2Fmois) ne sont pas indexées par Google. Je pense qu'il faudra en passer par de la génération dynamique côté serveur, ce qui se fait assez facilement avec Next.js et Vercel, cf. #1989

**Comment by johangirod** [comment]
Plus que la génération dynamique de page côté serveur, ce qu'il manque ici, ce sont des liens vers ces pages-là afin qu'elles soient d'avantage référencées. En effet, Google arrive déjà à indexer des pages avec des paramètres d'url correctement : 

https://www.google.com/search?q=simulation+imp%C3%B4t+auto-entrepreneur+5000+%E2%82%AC

Il faudrait : 
1. [ ]  Avoir des liens générés automatiquement qui contiennent un assemblage de situations commune, soit dans le sitemap, soit quelque part dans le footer ? On pourrait d'ailleurs se baser sur les tests de non regression pour les générer
1. [ ] Avoir un titre de la page qui s'adapte à la situation courante pour apparaître 
1. [ ] Avoir une logique pour filtrer les paramètres de situation qui ne doivent pas être référencés (pour éviter les erreurs lors de changements de nom de règles par exemple). 

Pour le point 3, il faudrait choisir quelles règles sont référencées : 
- uniquement celle avec un `identifiant court` (whitelist)
- ou toutes celles qui ne sont pas taguées comme `experimental` (blacklist)
